### PR TITLE
Improved parameter handling and documentation of continuous distributions

### PR DESCRIPTION
**Pull request by supreet11agrawal**
Improved parameter handling and documentation of continuous distributions
(till Dagum distribution). Also added test cases for these changes.
Entire crv_types has such shortcomings. Will add further commits to
change them as well

#### Brief description of what is fixed or changed
Documentation Change - Arcsine, Benini, Beta, Betaprime, Cauchy,
                                         Chi, ChiNoncentral, ChiSquared, Dagum
Parameter Checking - Betaprime, Cauchy, Chi, ChiNonCentral, ChiSquared, Dagum
#### Release Notes

* stats
  * Improved parameter handling by univariate continuous distributions
  * Corrected documentation for Cauchy distribution

### CODE REVIEW

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -349,10 +354,8 @@ def Beta(name, alpha, beta):
     Parameters
     ==========
 
-    alpha : real positive
-        A shape
-    beta : real positive
-        A shape
+    alpha : Real number, `\alpha > 0`, a shape
```
Comment (jksuom) [inline sympy/stats/crv_types.py]: I don't think that these should be changed. According to [numpydoc specifications](https://numpydoc.readthedocs.io/en/latest/format.html), the parameter description is expected to be on another line.

Comment (supreet11agrawal) [reply sympy/stats/crv_types.py]: Oh ok. I will do that. @jksuom should I apply these changes for other cases as well?(when original is not according to the specifications) like https://github.com/sympy/sympy/blob/b884ca44c955fe8979b721a3fbe967cbaeaf0134/sympy/stats/crv_types.py#L190

Comment (supreet11agrawal) [reply sympy/stats/crv_types.py]: I have not included these changes in this PR

Comment (jksuom) [reply sympy/stats/crv_types.py]: I think you could change originals that are not to the specification.

**Comment by jksuom** [comment]
Thanks, I think this can be merged. It is possible to continue later.

**Pull request merged**